ArduPlane/Plane.h

```diff
@@ -944,7 +944,7 @@ class Plane : public AP_Vehicle {
 #endif
 
     // commands.cpp
-    void set_guided_WP(const Location &loc);
+    void set_guided_WP(const Location &loc, const float radius = 0); // radius values: -1 = default (use param WP_LOITER_RAD), 0 = ignore (no-change), >0 is radius
```

Comment:
Why have such complicated semantics for the second parameter?

Wouldn't it be easier for the (two) callers to call `set_guided_radius` directly?

Reply:
I had it like that at one point but it didn't seem right that anyone could reach down into the guided mode class and set that property anytime. Seems better as an atomic-ish thing with a single entry point